user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Relation_learning/Adding_time_scoped_related_parameters_to_a_trend_graph.md

```diff
@@ -4,9 +4,9 @@ uid: Adding_time_scoped_related_parameters_to_a_trend_graph
 
 # Adding time-scoped related parameters to a trend graph
 
-From DataMiner 10.3.8/10.4.0 onwards<!--RN 36434 -->, a light bulb icon will be displayed when you select a time range on the trend graph of a parameter. If you want to know which other parameters are related to this parameter, based purely on the behavior during the selected time range, then you can click this icon to add or view related parameters. Even if multiple curves are displayed on the same trend graph, the light bulb always shows relations with one specific parameter, whose name is mentioned in the light bulb tooltip. Currently, the feature only proposes parameters from the same DataMiner element.
+From DataMiner 10.3.8/10.4.0 onwards<!--RN 36434 -->, a light bulb icon will be displayed when you select a time range on the trend graph of a parameter. If you want to know which other parameters are related to this parameter, based purely on the behavior during the selected time range, then you can click this icon to add or view related parameters. Even if multiple curves are displayed on the same trend graph, the light bulb always shows relations with one specific parameter, whose name is mentioned in the light bulb tooltip. The feature proposes parameters from the same DataMiner element as well as from other elements within the same service. Prior to DataMiner 10.4.11/10.5.0<!--RN 40658-->, the feature only proposes parameters from the same DataMiner element.
 
-You can for instance use this in case a parameter (e.g. the total available memory of a server) behaves oddly during a particular time range (e.g. a downward spike), in order to find out if other parameters of the same device also showed unusual behavior during the same time range.
+You can for instance use this in case a parameter (e.g. the total available memory of a server) behaves oddly during a particular time range (e.g. a downward spike), in order to find out if other parameters, either from the same device or from devices within the same service, also showed unusual behavior during the same time range.
```

Comment:
Maybe we should change "device" to "element" here, so we don't exclude elements that don't monitor an actual physical device.
```suggestion
You can for instance use this in case a parameter (e.g. the total available memory of a server) behaves oddly during a particular time range (e.g. a downward spike), in order to find out if other parameters, either from the same element or from elements within the same service, also showed unusual behavior during the same time range.
```